.github/workflows/test_on_push.yml

```diff
@@ -88,7 +92,7 @@ jobs:
       if: matrix.os == 'ubuntu-latest'
       run: tox -e examples
         
-    - name: Install and run coverage
+    - name: Run unit tests for GNU/Linux with Python 3.9 and generate coverage report
```

Comment:
Can we move this to line 78? Better to check the unit tests before integration tests and examples